src/adapter/src/coord.rs

```diff
@@ -1191,9 +1195,14 @@ impl<S: Append + 'static> Coordinator<S> {
     async fn advance_local_inputs(&mut self, advance_to: mz_repr::Timestamp) {
         self.advance_tables.insert(
             advance_to,
-            self.catalog
-                .entries()
-                .filter_map(|e| if e.is_table() { Some(e.id()) } else { None }),
+            self.catalog.entries().filter_map(|e| {
+                // TODO(andrioni): figure out if there's a way to let this not conflict with updates from storaged
```

Comment:
I don't understand what this TODO is worried about. What are the possible conflicts?

Reply:
This only works if noone else appends to the collection, otherwise it panics, as the upper is usually past what it expected it to be.

Reply:
This further suggests not making this a table then, as that's already what sources and bulitin logs are.